src/praisonai-agents/praisonaiagents/agent/agent.py

```
@@ -705,6 +705,13 @@ def __init__(
             Agent._configure_logging()
             Agent._logging_configured = True
 
+        # Auto-enable entry-point plugins when PRAISONAI_PLUGINS / config.toml requests it
+        try:
+            from ..plugins import maybe_enable_from_config
+            maybe_enable_from_config()
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)